LICENSE

```diff
@@ -175,3 +175,28 @@
       of your accepting any such warranty or additional liability.
 
    END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
```

Comment:
Not sure why pluginsync has re-added the appendix here. It's not part of the terms and conditions of the Apache license.